app/models/wakes/location.rb

```diff
@@ -13,7 +14,8 @@ def label
     path
   end
 
-  def url(protocol: 'http', host: ENV['DEFAULT_HOST'])
+  def url(protocol: 'http', host: nil)
+    host ||= self.host || ENV['DEFAULT_HOST']
```

Comment:
this is a rather oblique line of code.

Your goal, if I understand it, is to do something like `host || self.host || ENV['DEFAULT_HOST']`, right?

@gaganawhad do you know whether we really need to pass in the `host` parameter here? Do we ever use it?